plugins/ci/skills/manage-labels/manage_labels.py

```
@@ -0,0 +1,155 @@
+    for ctx in payload.get("hide_display_contexts") or []:
+        if ctx not in VALID_HIDE_CONTEXTS:
+            errs.append("invalid hide_display_contexts value %r (valid: %s)" % (ctx, ", ".join(VALID_HIDE_CONTEXTS)))
+    return errs
+
+
+def request(method, url, token, payload=None):
+    headers = {"Content-Type": "application/json", "Authorization": "Bearer %s" % token}
+    data = json.dumps(payload).encode("utf-8") if payload is not None else None
+    req = urllib.request.Request(url, data=data, headers=headers, method=method)
+    try:
+        with urllib.request.urlopen(req, timeout=30) as resp:
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)